tests/Java.Base-Tests/Java.Base/JavaVMFixture.cs

```diff
@@ -0,0 +1,27 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Diagnostics;
+using System.Linq;
+
+using Java.Interop;
+
+using Java.InteropTests;
+
+namespace Java.BaseTests {
+
+	partial class JavaVMFixture {
+
+		static partial void CreateJavaVM ()
+		{
+			var c = new TestJVM (
+					jars:           new[]{ "java.base-tests.jar" },
+					typeMappings:   new Dictionary<string, Type> () {
+						["crc64340ef73792c2a832/MyRunnable"] = typeof (MyRunnable),
```

Comment:
Is it worth putting the equivalent of `[Register("foo.MyRunnable")]` in case the CRC algorithm changes again?